### DISCUSSION TITLE
Run the Spirit of Competition event to celebrate the 2024 Summer Olympics.

### DISCUSSION THREAD

**Discussion by heyitsbench** [chromiecraft/chromiecraft#6035] [Suggestions]
The [Spirit of Competition](https://wowpedia.fandom.com/wiki/Spirit_of_Competition) event is a two-time (once during retail TBC, once during Classic TBC) in-game event celebrating the Summer Olympics during the years they occur. Since we are in TBC content and the Summer Olympics are approaching next year, I propose that CC run the event next year alongside the Olympics. After all, it's not like it'll need to be [implemented from scratch](https://github.com/azerothcore/azerothcore-wotlk/pull/16963). Plus it'll be a small lil incentive to get people to run battlegrounds during the event for the rewards.

It would probably require taking those rewards out of the catalogue since it would obviously cheapen the event, but not my call.

**Comment by heyitsbench** [comment]
The PR linked in this suggestion has now been merged, there is now nothing stopping CC from running this event.

Please? 🥺

**Comment by Peppersprayz** [comment]
Great idea

**Comment by minthes** [comment]
Seems fun

**Comment by 0mnimodus** [comment]
It would be great to take part in this

**Comment by andovcr** [comment]
The 2024 Olympics in Paris will take place:
Fri, Jul 26, 2024 – Sun, Aug 11, 2024

Any chance we could get a staff response prior to then?

**Comment by rakrak1775** [comment]
bench is right devs make it happen yesterday

**Comment by BeefSteaker** [comment]
Closing as accepted
